modules/samples/oauth2/playground2/src/main/webapp/oauth2-authorize-user.jsp

```diff
@@ -76,6 +81,12 @@
             .setResponseType(authzGrantType)
             .setScope(scope);
 
+    if (formPostMode) {
+        oAuthPKCEAuthenticationRequestBuilder.setParameter(OAuth2Constants.OAUTH2_RESPONSE_MODE, OAuth2Constants.OAUTH2_FORM_POST);
+    }
+
+
+//build the new response mode with form post
```

Comment:
Fix formatiing

Reply:
Fixed with https://github.com/wso2/product-is/pull/1375